content/overview/security/for-assessors.md

```diff
@@ -0,0 +1,61 @@
+---
+menu:
+  overview:
+    parent: technology
+title: Overview for assessors
+weight: 50
+aliases:
+  - /docs/intro/security/for-assessors
+  - /intro/security/for-assessors
+  - /overview/security/for-assessors/
+---
+
+## How cloud.gov works
+
+cloud.gov is a platform as a service (PaaS). This means customers provide the code for their app and we provide the infrastructure, operating system, and supporting technologies required to run the app. Three basic components undergird any cloud.gov app:
+
+1. Hardware and infrastructure-level software, including web servers, relational database servers, DNS routing services, and other systems the customer might need.
+1. A container providing the operating system and web server software (NGINX) and other basic software required to run any app. Apps are deployed into this container and isolated from other apps on the platform.
+1. Software components specific to the app the customer is launching, deployed along with the customer's app. These components, called buildpacks, provide programming languages and some additional dependencies required by the app. Though it's possible for customers to use their own buildpacks, we strongly encourage you to use the buildpacks we provide. That way you can benefit from our security updates (see below).
```

Comment:
"you" in this context is an assessor. Perhaps "we strongly encourage customers"?